data/presets/power/portal.json

```diff
@@ -0,0 +1,29 @@
+{
+    "icon": "temaki-power",
+    "fields": [
+        "ref",
+        "operator",
+        "structure",
```

Comment:
I don't think this is the right field for this. :wink:  The `structure` is the field for specifying bridge/tunnel/etc. on highways and similar features.  Here, only the tag actually has the `structure` key and the allowed values are `lattice`, `solid`, and `tubular`. We'd need a new field for this.

Reply:
Thanks for catching this! I added a `structure_power` field for this